lib/utils/nodeFieldIndices.mjs

```diff
@@ -0,0 +1,118 @@
+import getAtRuleParams from './getAtRuleParams.mjs';
+import getRuleSelector from './getRuleSelector.mjs';
+
+import { isObject, isString } from './validateTypes.mjs';
+
+/** @import {AtRule, Declaration, Rule} from 'postcss' */
+
+/**
+ * @param {AtRule} atRule
+ * @returns {number}
+ */
+export function atRuleParamIndex(atRule) {
+	let index = atRuleRawsAfterNameIndex(atRule);
+
+	if (atRule.raws.afterName) {
+		index += atRule.raws.afterName.length;
+	}
+
+	return index;
+}
+
+/**
+ * @param {AtRule} node
+ * @returns {number}
+ */
+export function atRuleRawsAfterIndex(node) {
```

Comment:
[suggestion] As well as other at-rule functions, renaming to `atRule` seems better in terms of consistency:

```suggestion
export function atRuleRawsAfterIndex(atRule) {
```